src/Stats.AzureCdnLogs.Common/Collect/OutputLogLine.cs

```diff
@@ -84,7 +84,22 @@ public static string Header
 
         public override string ToString()
         {
-            return $"{TimeStamp} {TimeTaken} {CIp} {FileSize} {SIp} {SPort} {ScStatus} {ScBytes} {CsMethod} {CsUriStem} - {RsDuration} {RsBytes} {CReferrer} {CUserAgent} {CustomerId} {XEc_Custom_1}";
+            return $"{TimeStamp} {TimeTaken} {CIp} {FileSize} {SIp} {SPort} {ScStatus} {ScBytes} {CsMethod} {CsUriStem} - {RsDuration} {RsBytes} {CReferrer} {Quote(CUserAgent)} {CustomerId} {Quote(XEc_Custom_1)}";
+        }
+
+        private static string Quote(string input)
+        {
+            if ( input.StartsWith("\"") && input.EndsWith("\"") )
```

Comment:
Need remove extra space here? 
```suggestion
            if (input.StartsWith("\"") && input.EndsWith("\""))
```